code/Test_definitions/Geofencing.feature

```diff
@@ -0,0 +1,114 @@
+
+
+@Geofencing 
+Feature: Camara Geofencing Subscriptions API ,0.2.0-rc3 Operations on subscriptions 
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in geofencing-subscriptions.yaml, version v0.2.0-rc3
+
+  Background: Common Geofencing Subscriptions setup
+    Given the resource "/geofencing-subscriptions/v0/subscriptions"  as geofencing Url                                                            |
```

Comment:
```suggestion
    Given the resource "/geofencing-subscriptions/v0.3/subscriptions"                                                           |
```
The version in path has to be updated to latest guidelines (also in the test plans for the other APIS)